portality/scripts/articles_invalid_url.py

```diff
@@ -0,0 +1,49 @@
+from portality import models
+from portality.core import app
+from portality.clcsv import UnicodeWriter
+import esprit
+import codecs
+import re
+
+INVALID_URLS = {
+    "query": {
+        "filtered": {
+            "filter": {
+                "exists": {"field": "index.fulltext"}
+            },
+            "query": {
+                "match_all": {}
+            }
+        }
+    },
+    "sort": [{
+        "in_doaj": "desc"
```

Comment:
Does this do the ones that are in_doaj: true first? Do we want to do the ones that aren't in the DOAJ at all?

Reply:
Yes, it does in_doaj: true first. And to be honest I don't know if we need one these that aren't in doaj at all so I just took all to check easily what is the scope of the issue.